Lib/asyncio/base_events.py

```diff
@@ -778,8 +786,11 @@ def _getaddrinfo_debug(self, host, port, family, type, proto, flags):
                          sock, host, port, transport, protocol)
         return transport, protocol
 
-    async def _create_connection_transport(self, sock, protocol_factory, ssl,
-                                           server_hostname, server_side=False):
+    async def _create_connection_transport(
+            self, sock, protocol_factory, ssl,
+            server_hostname, server_side=False,
+            ssl_handshake_timeout=constants.SSL_HANDSHAKE_TIMEOUT
```

Comment:
Move the `):` one line up: `ssl_handshake_timeout=constants.SSL_HANDSHAKE_TIMEOUT):`